### DISCUSSION TITLE
Channel Ten [TZ]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#452] [Show and tell]
### Channel ID

ChannelTen.tz

### Stream URL (optional)

https://dacastmmd.mmdlive.lldns.net/dacastmmd/0f5cd45bb20c4c2db122ac583d37a071/playlist.m3u8

### Notes (optional)

360p

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://dacastmmd.mmdlive.lldns.net/dacastmmd/0f5cd45bb20c4c2db122ac583d37a071/playlist.m3u8 (Offline)
<img width="721" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/b6402927-a0d2-42ee-9401-7529db98718c">
